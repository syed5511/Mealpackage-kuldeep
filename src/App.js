import React from "react";
import "bootstrap/dist/css/bootstrap.min.css";
import Footer from "./common_components/footer.js";
import FooterTerms from "./common_components/footer_terms.js";
import PackageButton from "./common_components/button.js";
import PackageSignup from "./common_components/package_signup.js";
import Signin from "./common_components/signin.js";
import FollowUs from "./common_components/followus.js";
import PricingPlan from "./common_components/pricing_plan.js";
import ConfirmAndPay from "./common_components/confirm_and_pay.js";
import Steppers from "./common_components/steppers.js";
import Skeleton from "./common_components/skeleton";
import Card from "./common_components/card";

import "./App.css";

function App() {
  return (
    <>
      {/* <Steppers /> */}
      {/* <ConfirmAndPay /> */}
      <PricingPlan />
      <Card>
        <Skeleton
          width="100%"
          height="100%"
          borderRadius="10px"
          className="rect"
        />
      </Card>
      <Skeleton className="circle" width="64px" />
      <Skeleton primaryColor="#ff9977" secondaryColor="#ff7777" width="20%" />
      <Skeleton
        className="circle"
        width="64px"
        primaryColor="#ff9977"
        secondaryColor="#ff7777"
      />
      {/* <Signin/> */}
      {/* <PackageSignup /> */}
      <FollowUs />
      <Footer />
      <FooterTerms />
    </>
  );
}

export default App;
