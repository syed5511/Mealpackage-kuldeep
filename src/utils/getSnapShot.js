import React from "react";
import { create, act } from "react-test-renderer";

const matchSnapShot = (json) => {
  expect(json).toMatchSnapshot();
};

const getSnapShot = (Component) => {
  let component;
  return (props = {}, type = "c") => {
    if (type === "c") {
      act(() => {
        component = create(<Component {...props} />);
      });
    } else {
      act(() => {
        component.update(<Component {...props} />);
      });
    }
    matchSnapShot(component.toJSON());
    return component;
  };
};

export default getSnapShot;
