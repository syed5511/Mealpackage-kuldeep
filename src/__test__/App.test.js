import App from "../App";
import getSnapShot from "../utils/getSnapShot";

test("App js snapshot", () => {
  getSnapShot(App);
});
