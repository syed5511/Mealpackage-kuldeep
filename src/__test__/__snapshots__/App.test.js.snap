// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App js snapshot 1`] = `
Array [
  <header
    className="sc-fznJRM ieAZXc"
  >
    <div
      className="sc-fzqARJ iOekWf"
    >
      <div
        className="sc-fzoiQi dAagjF"
      >
        Logo
      </div>
      <div
        className="sc-fzqNqU eZYNRh"
      >
        <ul
          className="sc-AxjAm egATdG"
        >
          <li
            className="sc-AxirZ jIOcLI"
            onClick={[Function]}
          >
            <a
              aria-current="page"
              className="active"
              href="/"
              onClick={[Function]}
              style={Object {}}
            >
              ON MENU
            </a>
          </li>
          <li
            className="sc-AxirZ jIOcLI"
            onClick={[Function]}
          >
            <a
              aria-current={null}
              href="/pricing"
              onClick={[Function]}
            >
              PRICING
            </a>
          </li>
        </ul>
        <div
          className="sc-AxiKw cQiIBL"
        >
          <div
            className="popover"
          >
            <div
              className="sc-AxhCb kYwVIA"
              onClick={[Function]}
              onMouseOver={[Function]}
            >
              <div
                className="sc-fznZeY cUOZzi"
              >
                <img
                  alt=""
                  className="sc-fzqBZW eksQHo"
                  src="/static/images/pin.png"
                />
                <div
                  className="sc-fzokOt hYUweD"
                >
                  <div
                    className="sc-fzqNJr kuikfi"
                  >
                    We don't have delivery service to
                  </div>
                  <div
                    className="sc-fzoyAV dYJGmn"
                  >
                    Skokie - 606060
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-fzoNJl goAsPW"
        onClick={[Function]}
      >
        <div
          className="sc-fzoXWK bmdwhh"
        />
        <div
          className="sc-fzoXWK cMNQfC"
        />
        <div
          className="sc-fzoXWK cdkarF"
        />
      </div>
    </div>
    <div
      className="sc-fzqARJ sc-fzoyTs kuYyTT"
    >
      <div
        className="sc-fznKkj eimxmm"
      >
        <div
          className="sc-AxiKw cQiIBL"
        >
          <div
            className="popover"
          >
            <div
              className="sc-AxhCb kYwVIA"
              onClick={[Function]}
              onMouseOver={[Function]}
            >
              <div
                className="sc-fzpans clgpyt"
              >
                <img
                  alt=""
                  className="sc-fznyAO aqOFJ loggedIn"
                  src="/static/images/restaurant.png"
                />
                <div
                  className="sc-fzplWN dwboMu"
                >
                  Hello Syed
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-fznKkj eimxmm small"
      >
        <div
          className="sc-AxhUy dXLonu"
        >
          <div
            className="sc-AxgMl iqBGmz"
          >
            <ul
              className="sc-AxheI dTtpEJ"
            >
              <li
                className="sc-Axmtr haTalS"
              >
                Order History
              </li>
              <li
                className="sc-Axmtr haTalS"
              >
                Delivery Address
              </li>
              <li
                className="sc-Axmtr haTalS"
              >
                Subscription
              </li>
            </ul>
            <button
              className="sc-AxmLO kjIQHd"
            >
              Logout
            </button>
          </div>
          <hr
            className="sc-fzozJi ekRuVc"
          />
          <div
            className="sc-fzoLsD iNRIDG"
          >
            Available Plates 
            <span>
              5
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className="sc-fzpmMD kcxMOg"
    >
      <img
        alt=""
        className="sc-fznMAR fYiUZr"
        onClick={[Function]}
        src="/static/images/cross.png"
      />
      <div
        className="sc-fznxKY ijdIKE"
      >
        Hello Syed!
      </div>
      <ul
        className="sc-AxjAm egATdG"
      >
        <li
          className="sc-AxirZ jIOcLI"
          onClick={[Function]}
        >
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
            style={Object {}}
          >
            ON MENU
          </a>
        </li>
        <li
          className="sc-AxirZ jIOcLI"
          onClick={[Function]}
        >
          <a
            aria-current={null}
            href="/pricing"
            onClick={[Function]}
          >
            PRICING
          </a>
        </li>
      </ul>
      <div
        className="sc-fznKkj eimxmm"
      >
        <div
          className="sc-AxiKw cQiIBL"
        >
          <div
            className="popover"
          >
            <div
              className="sc-AxhCb kYwVIA"
              onClick={[Function]}
              onMouseOver={[Function]}
            >
              <div
                className="sc-fzpans clgpyt"
              >
                <img
                  alt=""
                  className="sc-fznyAO aqOFJ loggedIn"
                  src="/static/images/restaurant.png"
                />
                <div
                  className="sc-fzplWN dwboMu"
                >
                  Hello Syed
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-fznKkj eimxmm small"
      >
        <div
          className="sc-AxhUy dXLonu"
        >
          <div
            className="sc-AxgMl iqBGmz"
          >
            <ul
              className="sc-AxheI dTtpEJ"
            >
              <li
                className="sc-Axmtr haTalS"
              >
                Order History
              </li>
              <li
                className="sc-Axmtr haTalS"
              >
                Delivery Address
              </li>
              <li
                className="sc-Axmtr haTalS"
              >
                Subscription
              </li>
            </ul>
            <button
              className="sc-AxmLO kjIQHd"
            >
              Logout
            </button>
          </div>
          <hr
            className="sc-fzozJi ekRuVc"
          />
          <div
            className="sc-fzoLsD iNRIDG"
          >
            Available Plates 
            <span>
              5
            </span>
          </div>
        </div>
      </div>
    </div>
  </header>,
  <div
    className="sc-fznBMq fYtPWH"
  >
    <div
      className="sc-fzqKxP dtcFuf"
    >
      <div
        className="sc-pANHa dDSvYk"
      >
        <div
          className="sc-pIJJz eWNOVt"
        />
        <div
          className="sc-pRFjI glBhHD"
        >
          <div
            className="sc-pZBmh ILuKv"
          >
            Delivery before
          </div>
          <div
            className="sc-pZBmh sc-oTbqq gbjhYO"
          >
            01:30 pm
          </div>
        </div>
      </div>
      <div
        id="slot-container"
      >
        <div
          className="chip active"
          onClick={[Function]}
        >
          <div
            className="day"
          >
            Tuesday
          </div>
          <div
            className="month"
          >
            04
             
            August
          </div>
        </div>
        <div
          className="chip"
          onClick={[Function]}
        >
          <div
            className="day"
          >
            Wednesday
          </div>
          <div
            className="month"
          >
            05
             
            August
          </div>
        </div>
        <div
          className="chip"
          onClick={[Function]}
        >
          <div
            className="day"
          >
            Thursday
          </div>
          <div
            className="month"
          >
            06
             
            August
          </div>
        </div>
        <div
          className="chip"
          onClick={[Function]}
        >
          <div
            className="day"
          >
            Friday
          </div>
          <div
            className="month"
          >
            07
             
            August
          </div>
        </div>
        <div
          className="chip"
          onClick={[Function]}
        >
          <div
            className="day"
          >
            Saturday
          </div>
          <div
            className="month"
          >
            08
             
            August
          </div>
        </div>
        <div
          className="chip"
          onClick={[Function]}
        >
          <div
            className="day"
          >
            Sunday
          </div>
          <div
            className="month"
          >
            09
             
            August
          </div>
        </div>
        <div
          className="chip"
          onClick={[Function]}
        >
          <div
            className="day"
          >
            Monday
          </div>
          <div
            className="month"
          >
            10
             
            August
          </div>
        </div>
      </div>
      <div
        className="sc-paXsP kRdSbc"
      >
        <img
          alt=""
          className="sc-pjSSY beMxEI"
          src="/static/images/alert.gif"
        />
        <div
          className="sc-prOVx gElGAV"
        >
          You're meal plan allow you to add 2 main course
        </div>
      </div>
    </div>
    <h2
      className="sc-fzonZV ftMqRU"
    >
      Main Course
    </h2>
    <div
      className="sc-fznNvL iSEFHf"
    >
      <div
        className="sc-fzqyOu cXkKeC"
      >
        <div
          className="sc-fzqzlV jZSzWN"
        >
          <div
            className="sc-fzqLLg bnQYa-d"
          />
          <div
            className="sc-fzoxnE dAyMmu"
          >
            <div
              className="sc-fzoMdx exCRRL"
            >
              <h3
                className="sc-fzoWqW cJKlPL"
              >
                Chicken Chicken Chicken
              </h3>
              <p
                className="sc-fzplgP jNPGH"
              >
                with red chili sauce
              </p>
              <div
                className="sc-fznzqM bPJlig"
              >
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
              </div>
            </div>
            <button
              className="sc-fznYue iILixL"
            >
              +
            </button>
          </div>
        </div>
        <div
          className="sc-fzqzlV jZSzWN"
        >
          <div
            className="sc-fzqLLg bnQYa-d"
          />
          <div
            className="sc-fzoxnE dAyMmu"
          >
            <div
              className="sc-fzoMdx exCRRL"
            >
              <h3
                className="sc-fzoWqW cJKlPL"
              >
                Chicken Chicken Chicken
              </h3>
              <p
                className="sc-fzplgP jNPGH"
              >
                with red chili sauce
              </p>
              <div
                className="sc-fznzqM bPJlig"
              >
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
              </div>
            </div>
            <button
              className="sc-fznYue iILixL"
            >
              +
            </button>
          </div>
        </div>
        <div
          className="sc-fzqzlV jZSzWN"
        >
          <div
            className="sc-fzqLLg bnQYa-d"
          />
          <div
            className="sc-fzoxnE dAyMmu"
          >
            <div
              className="sc-fzoMdx exCRRL"
            >
              <h3
                className="sc-fzoWqW cJKlPL"
              >
                Chicken Chicken Chicken
              </h3>
              <p
                className="sc-fzplgP jNPGH"
              >
                with red chili sauce
              </p>
              <div
                className="sc-fznzqM bPJlig"
              >
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
              </div>
            </div>
            <button
              className="sc-fznYue iILixL"
            >
              +
            </button>
          </div>
        </div>
        <div
          className="sc-fzqzlV jZSzWN"
        >
          <div
            className="sc-fzqLLg bnQYa-d"
          />
          <div
            className="sc-fzoxnE dAyMmu"
          >
            <div
              className="sc-fzoMdx exCRRL"
            >
              <h3
                className="sc-fzoWqW cJKlPL"
              >
                Chicken Chicken Chicken
              </h3>
              <p
                className="sc-fzplgP jNPGH"
              >
                with red chili sauce
              </p>
              <div
                className="sc-fznzqM bPJlig"
              >
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
              </div>
            </div>
            <button
              className="sc-fznYue iILixL"
            >
              +
            </button>
          </div>
        </div>
        <div
          className="sc-fzqzlV jZSzWN"
        >
          <div
            className="sc-fzqLLg bnQYa-d"
          />
          <div
            className="sc-fzoxnE dAyMmu"
          >
            <div
              className="sc-fzoMdx exCRRL"
            >
              <h3
                className="sc-fzoWqW cJKlPL"
              >
                Chicken Chicken Chicken
              </h3>
              <p
                className="sc-fzplgP jNPGH"
              >
                with red chili sauce
              </p>
              <div
                className="sc-fznzqM bPJlig"
              >
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
                <div
                  className="sc-fznOgF hhcRYr"
                />
              </div>
            </div>
            <button
              className="sc-fznYue iILixL"
            >
              +
            </button>
          </div>
        </div>
      </div>
      <div
        className="sc-fzocqA gbTxyD"
      >
        <div
          className="sc-fzobTh iIxmWM"
        >
          <img
            alt=""
            className="sc-fzooss ePtIhq"
            src="/static/images/cooking.png"
          />
          <h3
            className="sc-fzqyvX hbvfGc"
          >
            Your Cart is Empty
          </h3>
          <p
            className="sc-fzqKVi lciyaR"
          >
            Good food is always cooking! Go ahead, order some yummy items from the menu.
          </p>
        </div>
      </div>
    </div>
    <div
      className="sc-fznNvL iSEFHf dessert"
    >
      <h2
        className="sc-fzonZV ftMqRU dessert"
      >
        Desserts & Salads
      </h2>
      <div
        className="sc-fzqyOu cXkKeC dessert"
      >
        <div
          className="sc-fzonjX gCEQTb"
        >
          <div
            className="sc-fzqzEs jEodoa"
          />
          <div
            className="sc-fzqOul ezcMbW"
          >
            <div
              className="sc-fzoCCn iwCOdk"
            >
              <h3
                className="sc-fzoPby ifPOIl"
              >
                Chicken Chicken Chicken
              </h3>
              <p
                className="sc-fzpdbB eECnNY"
              >
                with red chili sauce
              </p>
              <div
                className="sc-fzppAM frvfAA"
              >
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
              </div>
            </div>
            <button
              className="sc-fznNTe igDAju"
            >
              +
            </button>
          </div>
        </div>
        <div
          className="sc-fzonjX gCEQTb"
        >
          <div
            className="sc-fzqzEs jEodoa"
          />
          <div
            className="sc-fzqOul ezcMbW"
          >
            <div
              className="sc-fzoCCn iwCOdk"
            >
              <h3
                className="sc-fzoPby ifPOIl"
              >
                Chicken Chicken Chicken
              </h3>
              <p
                className="sc-fzpdbB eECnNY"
              >
                with red chili sauce
              </p>
              <div
                className="sc-fzppAM frvfAA"
              >
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
              </div>
            </div>
            <button
              className="sc-fznNTe igDAju"
            >
              +
            </button>
          </div>
        </div>
        <div
          className="sc-fzonjX gCEQTb"
        >
          <div
            className="sc-fzqzEs jEodoa"
          />
          <div
            className="sc-fzqOul ezcMbW"
          >
            <div
              className="sc-fzoCCn iwCOdk"
            >
              <h3
                className="sc-fzoPby ifPOIl"
              >
                Chicken Chicken Chicken
              </h3>
              <p
                className="sc-fzpdbB eECnNY"
              >
                with red chili sauce
              </p>
              <div
                className="sc-fzppAM frvfAA"
              >
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
              </div>
            </div>
            <button
              className="sc-fznNTe igDAju"
            >
              +
            </button>
          </div>
        </div>
        <div
          className="sc-fzonjX gCEQTb"
        >
          <div
            className="sc-fzqzEs jEodoa"
          />
          <div
            className="sc-fzqOul ezcMbW"
          >
            <div
              className="sc-fzoCCn iwCOdk"
            >
              <h3
                className="sc-fzoPby ifPOIl"
              >
                Chicken Chicken Chicken
              </h3>
              <p
                className="sc-fzpdbB eECnNY"
              >
                with red chili sauce
              </p>
              <div
                className="sc-fzppAM frvfAA"
              >
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
              </div>
            </div>
            <button
              className="sc-fznNTe igDAju"
            >
              +
            </button>
          </div>
        </div>
        <div
          className="sc-fzonjX gCEQTb"
        >
          <div
            className="sc-fzqzEs jEodoa"
          />
          <div
            className="sc-fzqOul ezcMbW"
          >
            <div
              className="sc-fzoCCn iwCOdk"
            >
              <h3
                className="sc-fzoPby ifPOIl"
              >
                Chicken Chicken Chicken
              </h3>
              <p
                className="sc-fzpdbB eECnNY"
              >
                with red chili sauce
              </p>
              <div
                className="sc-fzppAM frvfAA"
              >
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
              </div>
            </div>
            <button
              className="sc-fznNTe igDAju"
            >
              +
            </button>
          </div>
        </div>
        <div
          className="sc-fzonjX gCEQTb"
        >
          <div
            className="sc-fzqzEs jEodoa"
          />
          <div
            className="sc-fzqOul ezcMbW"
          >
            <div
              className="sc-fzoCCn iwCOdk"
            >
              <h3
                className="sc-fzoPby ifPOIl"
              >
                Chicken Chicken Chicken
              </h3>
              <p
                className="sc-fzpdbB eECnNY"
              >
                with red chili sauce
              </p>
              <div
                className="sc-fzppAM frvfAA"
              >
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
                <div
                  className="sc-fznBtT ehidnX"
                />
              </div>
            </div>
            <button
              className="sc-fznNTe igDAju"
            >
              +
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="sc-fzoCUK bzNzly"
    >
      <div
        className="sc-fzoOEf cVfVDi"
      />
      <div
        className="sc-fzpdyU iTuInR"
      >
        <div
          className="sc-fzppip eZDNFl"
          onClick={[Function]}
        >
          Done
        </div>
        <div
          className="sc-pAZqv JYiM"
        >
          <img
            alt=""
            className="sc-qQmou eaprUK"
            src="/static/images/cooking.png"
          />
          <p
            className="sc-qYiqT jTeWaz"
          >
            Good food is always cooking! Go ahead, order some yummy items from the menu.
          </p>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="followus_container"
  >
    <div
      className="row"
      style={
        Object {
          "height": "100%",
        }
      }
    >
      <div
        className="col-12 followus_content"
      >
        <div
          className="row"
        >
          <div
            className="col-6 followus_left"
          >
            <div
              className="offset-lg-3 col-lg-9 col-md-12"
            >
              <div
                className="row"
              >
                <div
                  className="col-md-9"
                >
                  <div
                    className="whats_coockin"
                  >
                    Discover What's 
                    <span
                      className="cookin"
                    >
                      Cookin'
                    </span>
                  </div>
                  <form
                    className=""
                  >
                    <div
                      className="form-row"
                    >
                      <div
                        className="margin_auto form-group col-md-12"
                      >
                        <input
                          className="text_input subscribe_input form-control"
                          id="validationFormik03"
                          name="email"
                          placeholder="Email Address"
                          type="text"
                        />
                      </div>
                    </div>
                  </form>
                  <div
                    className="row"
                  >
                    <div
                      className="col-12 subscribe_follow_line"
                    >
                      Sign up for offers, recipes, news & more
                    </div>
                  </div>
                </div>
                <div
                  className="col-md-3 p-0 subscribe_container"
                >
                  <button
                    className="subscribe_button btn btn-primary"
                    disabled={false}
                    type="button"
                  >
                    Subscribe
                  </button>
                </div>
              </div>
            </div>
          </div>
          <div
            className="col-6 followus_right"
          >
            <div
              className="row"
            >
              <div
                className="col-md-10 offset-md-1"
              >
                <h6
                  className="follow_us_heading"
                >
                  Follow Us
                </h6>
                <a
                  href="#"
                >
                  <img
                    src="svg/facebook.svg"
                    width="50"
                  />
                </a>
                <a
                  href="#"
                >
                  <img
                    src="svg/twitter.svg"
                    width="50"
                  />
                </a>
                <a
                  href="#"
                >
                  <img
                    src="svg/instagram.svg"
                    width="50"
                  />
                </a>
                <a
                  href="#"
                >
                  <img
                    src="svg/pinterest.svg"
                    width="50"
                  />
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <footer
    class="footer py-8 py-md-11 @@classList"
  >
    <div
      class="container-lg"
    >
      <div
        class="row"
      >
        <div
          class="col-md-4"
        >
          <h2
            class="font-family-serif text-white mb-1"
          >
            Goodkit
          </h2>
          <p
            class="text-white-60"
          >
            A better way to build.
          </p>
        </div>
        <div
          class="col-1 col-md"
        >
          <h6
            class="text-uppercase text-white mb-3 mb-md-5"
          >
            Services
          </h6>
          <ul
            class="list-unstyled"
          >
            <li
              class="mb-3"
            >
              <a
                class="text-white-60"
                href="#!"
              >
                Help Center
              </a>
            </li>
            <li
              class="mb-3"
            >
              <a
                class="text-white-60"
                href="#!"
              >
                Press
              </a>
            </li>
            <li>
              <a
                class="text-white-60"
                href="#!"
              >
                About us
              </a>
            </li>
          </ul>
        </div>
        <div
          class="col-1 col-md"
        >
          <h6
            class="text-uppercase text-white mb-3 mb-md-5"
          >
            Products
          </h6>
          <ul
            class="list-unstyled"
          >
            <li
              class="mb-3"
            >
              <a
                class="text-white-60"
                href="#!"
              >
                Homepage
              </a>
            </li>
            <li
              class="mb-3"
            >
              <a
                class="text-white-60"
                href="#!"
              >
                Pages
              </a>
            </li>
            <li
              class="mb-3"
            >
              <a
                class="text-white-60"
                href="#!"
              >
                Legal
              </a>
            </li>
            <li>
              <a
                class="text-white-60"
                href="#!"
              >
                Terms
              </a>
            </li>
          </ul>
        </div>
        <div
          class="col-1 col-md"
        >
          <h6
            class="text-uppercase text-white mb-3 mb-md-5"
          >
            Services
          </h6>
          <ul
            class="list-unstyled"
          >
            <li
              class="mb-3"
            >
              <a
                class="text-white-60"
                href="#!"
              >
                Help Center
              </a>
            </li>
            <li
              class="mb-3"
            >
              <a
                class="text-white-60"
                href="#!"
              >
                Press
              </a>
            </li>
            <li>
              <a
                class="text-white-60"
                href="#!"
              >
                About us
              </a>
            </li>
          </ul>
        </div>
        <div
          class="col-1 col-md"
        >
          <h6
            class="text-uppercase text-white mb-3 mb-md-5"
          >
            Connect
          </h6>
          <ul
            class="list-unstyled"
          >
            <li
              class="mb-3"
            >
              <a
                class="text-white-60"
                href="#!"
              />
            </li>
            <li
              class="mb-3"
            >
              <a
                class="text-white-60"
                href="#!"
              />
            </li>
            <li
              class="mb-3"
            >
              <a
                class="text-white-60"
                href="#!"
              />
            </li>
            <li>
              <a
                class="text-white-60"
                href="#!"
              />
            </li>
          </ul>
        </div>
      </div>
    </div>
  </footer>,
  <div
    className="footer_terms"
  >
    <div
      className="row cover_terms"
    >
      <div
        className="col-1"
      />
      <div
        className="col-5 llc"
      >
        @ company name, LLC 2020 | A better way to eat healthy & fresh
      </div>
      <div
        className="col-1"
      />
      <div
        className="col-4 llc"
      >
        Do Not "Sell My Info"  
        <span>
          <a
            href="#"
          >
            Privacy Policy 
          </a>
        </span>
        <span>
          <a
            href="#"
          >
             Terms of Use
          </a>
        </span>
      </div>
    </div>
  </div>,
]
`;
